### PR TITLE
lsifstore: update testdata to include API docs

### PR DESCRIPTION
**Pull request by slimsag**
@efritz this was quite a trip.. what I had said in #26771 did not work at all, and I ended up wasting all of last Friday figuring this out in #26830 - I am pretty sure this is all good and correct now, PTAL.

### CODE REVIEW

File: enterprise/internal/codeintel/stores/lsifstore/helpers_test.go [hunk 1]
```diff
@@ -9,7 +10,7 @@ import (
 	"github.com/sourcegraph/sourcegraph/internal/observation"
 )
 
-const testBundleID = 39162
+const testBundleID = 1
```
Comment (slimsag) [inline enterprise/internal/codeintel/stores/lsifstore/helpers_test.go:13]: I am suspicious of how we generated this bundle in the first place? We must have hand-picked a bundle out of the DB? Regardless, with a fresh Sourcegraph DB the ID 1 is correct.

File: enterprise/internal/codeintel/stores/lsifstore/helpers_test.go [hunk 2]
```diff
@@ -29,12 +30,25 @@ func populateTestStore(t testing.TB) *Store {
...
+	var withoutComments []byte
+	for _, line := range bytes.Split(contents, []byte{'\n'}) {
+		if string(line) == "" || bytes.HasPrefix(line, []byte("--")) {
 			continue
 		}
+		withoutComments = append(withoutComments, line...)
+		withoutComments = append(withoutComments, '\n')
+	}
 
-		if _, err := tx.Exec(line); err != nil {
```
Comment (slimsag) [inline enterprise/internal/codeintel/stores/lsifstore/helpers_test.go:37]: Below here I switch us to properly execute statements separated by `;\n`, not `\n`, as string literals may span multiple lines (particularly true for API docs.)

File: enterprise/internal/codeintel/stores/lsifstore/ranges_test.go [hunk 3]
```diff
@@ -64,7 +64,8 @@ func TestDatabaseRanges(t *testing.T) {
 					{DumpID: testBundleID, Path: "protocol/writer.go", Range: newRange(150, 9, 150, 15)},
 					{DumpID: testBundleID, Path: "protocol/writer.go", Range: newRange(155, 9, 155, 15)},
 				},
-				HoverText: "```go\ntype Writer struct\n```\n\n---\n\nWriter emits vertices and edges to the underlying writer. This struct will guarantee that unique identifiers are generated for each element.\n\n---\n\n```go\nstruct {\n    w Writer\n    addContents bool\n    id int\n    numElements int\n}\n```",
+				HoverText:           "```go\ntype Writer struct\n```\n\n---\n\nWriter emits vertices and edges to the underlying writer. This struct will guarantee that unique identifiers are generated for each element. \n\n\n\n---\n\n```go\nstruct {\n    w Writer\n    addContents bool\n    id int\n    numElements int\n}\n```",
```
Comment (slimsag) [inline enterprise/internal/codeintel/stores/lsifstore/ranges_test.go:67]: ` \n\n` added

File: enterprise/internal/codeintel/stores/lsifstore/packages_test.go [hunk 4]
```diff
@@ -15,14 +15,14 @@ func TestDatabasePackageInformation(t *testing.T) {
...
+	if actual, exists, err := store.PackageInformation(context.Background(), testBundleID, "protocol/protocol.go", "114"); err != nil {
 		t.Fatalf("unexpected error %s", err)
 	} else if !exists {
 		t.Errorf("no package information")
 	} else {
 		expected := precise.PackageInformationData{
-			Name:    "github.com/sourcegraph/lsif-go",
-			Version: "v0.0.0-ad3507cbeb18",
+			Name:    "https://github.com/sourcegraph/lsif-go",
+			Version: "ad3507cbeb18",
```
Comment (efritz) [inline enterprise/internal/codeintel/stores/lsifstore/packages_test.go:25]: Did this really change?

File: enterprise/internal/codeintel/stores/lsifstore/testdata/README.md [hunk 5]
```diff
@@ -14,25 +14,4 @@ To regenerate `lsif-go@foo.sql` you will need to:
 pg_dump --data-only  --format plain --inserts --dbname sg --table 'lsif_data_*' --file test.sql
 ```
```
Comment (slimsag) [inline enterprise/internal/codeintel/stores/lsifstore/testdata/README.md:15]: These are the exact instructions above that I followed to produce this change. Also note that the below was deleted - that is not needed and was the result of me not knowing I needed to update `const testBundleID = 39162`

File: enterprise/internal/codeintel/stores/lsifstore/packages_test.go (hunk 4 above)
Comment (slimsag) [reply enterprise/internal/codeintel/stores/lsifstore/packages_test.go:25]: Yes, it absolutely did. You can reproduce it following the same steps in the README

I'm not sure why, though

File: enterprise/internal/codeintel/stores/lsifstore/testdata/README.md (hunk 5 above)
Comment (slimsag) [reply enterprise/internal/codeintel/stores/lsifstore/testdata/README.md:15]: Also noting: I used the latest lsif-go release binary (`v1.6.7` tag) to index the same `lsif-go` revision as described in the prior filename, `ad3507cb`

File: enterprise/internal/codeintel/stores/lsifstore/packages_test.go (hunk 4 above)
Comment (efritz) [reply enterprise/internal/codeintel/stores/lsifstore/packages_test.go:25]: Ok this commit is super old and before any tags.

**Review by efritz — APPROVED** [review]

**Pull request merged**